README.md

```diff
@@ -150,13 +150,12 @@ pairs as shown below:
 
    ![function code](https://cloud.githubusercontent.com/assets/12450298/12537413/7374470c-c2b6-11e5-8e3a-9baaa99c06aa.png)
 
-  5. Zip up this file by typing the following into the command line. The command consists of the first filename which is the zip file you want to create _(call it whatever you like .zip)_ followed by the files you want to zip up. In our example you can see the name of the ```.js``` file we created earlier:
-
-   `$ zip -r hello-world.zip hello-world.js`
-
-    You should now be able to see a ```.ZIP``` file alongside your ```.js``` file.  
-    **NOTE: If your function has any dependencies then you must include your ```node_modules``` file within your .ZIP file. Simply add ```node_modules``` after the files you wish to zip up!**
+5. Zip up this file by typing the following into the command line. The command consists of the first filename which is the zip file you want to create _(call it whatever you like .zip)_ followed by the files you want to zip up. In our example you can see the name of the ```.js``` file we created earlier:
+   
+ &nbsp; &nbsp; &nbsp; &nbsp; `$ zip -r hello-world.zip hello-world.js`
```

Comment:
There is a better way, using child-indicating 4-space-prefixed lines. The trick is, you have to enclose code with triple-backticks in mult-line format.